scripts/OIMS2fits.py

```diff
@@ -58,11 +60,11 @@ def compute_beam_pattern(az, alt, corr=corrFnc):
 def pbcorroims(header,imSize,chan):
     pScale = header['pixel_size']
     sRad   = 360.0/pScale/numpy.pi / 2
-    w = WCS(naxis=2)
-    w.wcs.crpix = [imSize/2 + 1 + 0.5 * ((imSize+1)%2),imSize/2 + 1 + 0.5 * ((imSize+1)%2)]
-    w.wcs.cdelt = numpy.array([-360.0/(2*sRad)/numpy.pi, 360.0/(2*sRad)/numpy.pi])
-    w.wcs.crval = [header['center_ra'], header['center_dec']]
-    w.wcs.ctype = ["RA---SIN", "DEC--SIN"]
+    if station==b'LWASV':
+        # SV has special correction factors to improve positionsQ
+        w = getSVwcs(header, imSize)
+    else:
+        w = getGENERICwcs(header, imSize)
```

Comment:
`station` isn't defined inside (or passed into) this function.

Reply:
Maybe we also need to update `test_OIMS2fits.py` to try some of the other options.

Reply:
Yes, looks like it would be good to add the following:
1. LWA-NA test data
2. pbcorr options (others missed?)

Reply:
Oh, and when I check the pbcorr in my transient searches it doesn't seem to be numerically stable. very odd. We should chat about it

Reply:
Maybe diff and index if there are enough entries in the test data set?

Reply:
I've added tests to cover all the options. Still no LWA-NA test data. I'm concerned if I add more data to the test directory it will become unwieldy. It's already a lot to pull from github

Reply:
There's NA data now for "only" an extra 17 MB.